background.ts

```diff
@@ -148,7 +161,7 @@ Electron.app.on('window-all-closed', () => {
 Electron.app.on('activate', () => {
   // On macOS it's common to re-create a window in the app when the
   // dock icon is clicked and there are no other windows open.
-  window.openPreferences();
+  window.openPreferences(false);
```

Comment:
Why would you not restore the window when the user has asked for it? I'm confused.

Reply:
As noted before, restoring the window fixes a bug.